plugins/ssrmatching/ssrmatching.ml4

```diff
@@ -12,8 +12,8 @@
  * we thus save the lexer to restore it at the end of the file *)
 let frozen_lexer = CLexer.get_keyword_state () ;;
 
-(*i camlp4use: "pa_extend.cmo" i*)
-(*i camlp4deps: "grammar/grammar.cma" i*)
+(*i camlp5use: "pa_extend.cmo" i*)
+(*i camlp5deps: "grammar/grammar.cma" i*)
```

Comment:
Can someone confirm that these two lines can just be removed? Or only the second one?